cmd/notation/policy/import.go

```diff
@@ -85,17 +87,40 @@ func runImport(command *cobra.Command, opts importOpts) error {
 			}
 		}
 	} else {
-		fmt.Fprintln(os.Stderr, "Warning: existing trust policy configuration file will be overwritten")
+		fmt.Fprintln(os.Stderr, "Warning: existing OCI trust policy file will be overwritten")
 	}
 
 	// write
-	policyPath, err := dir.ConfigFS().SysPath(dir.PathTrustPolicy)
+	policyPath, err := dir.ConfigFS().SysPath(dir.PathOCITrustPolicy)
 	if err != nil {
-		return fmt.Errorf("failed to obtain path of trust policy file: %w", err)
+		return fmt.Errorf("failed to obtain path of OCI trust policy file: %w", err)
 	}
 	if err = osutil.WriteFile(policyPath, policyJSON); err != nil {
-		return fmt.Errorf("failed to write trust policy file: %w", err)
+		return fmt.Errorf("failed to write OCI trust policy file: %w", err)
+	}
+	// user has confirmed to overwrite the existing trust policy file, clean
+	// the old trust policy file if exists
+	if err := cleanOldTrustPolicy(); err != nil {
+		fmt.Fprintf(os.Stderr, "Warning: failed to clean old trust policy file: %s\n", err)
+	} else {
+		fmt.Fprintln(os.Stdout, "Cleaned old trust policy file.")
 	}
-	_, err = fmt.Fprintln(os.Stdout, "Trust policy configuration imported successfully.")
+
+	_, err = fmt.Fprintf(os.Stdout, "Successfully imported OCI trust policy file to %s.\n", policyPath)
 	return err
 }
+
+// cleanOldTrustPolicy removes the old trust policy file if exists.
+func cleanOldTrustPolicy() error {
```

Comment:
Let's make it clearer:
```suggestion
// deleteOldTrustPolicyFile deletes the old trust policy file `trustpolicy.json` if exists.
func deleteOldTrustPolicyFile() error {
```

Reply:
Updated.